storage/redis/redisnamespace/redisnamespace.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package redisnamespace
+
+import "strconv"
+
+var dbs = map[string]int{
```

Comment:
Why define these here when only storj-sim needs them?

Reply:
good point, I started them for general use, but yeah only storj-sim is using it atm